source/locales/es/LC_MESSAGES/the_need_for.po

```diff
@@ -0,0 +1,473 @@
+msgstr ""
+
+#: ../../source/the_need_for.rst:21
+msgid "Where do ecosystem services originate and where are they consumed?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:22
+msgid ""
+"How does a proposed forestry management plan affect biodiversity, water "
+"quality, and recreation?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:23
+msgid ""
+"What kinds of coastal management and fishery policies will yield the best"
+" returns for sustainable fisheries, shoreline protection, and recreation?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:24
+msgid ""
+"Which parts of a watershed provide the greatest carbon sequestration, "
+"biodiversity, and tourism values?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:25
+msgid ""
+"Where would reforestation achieve the greatest downstream water quality "
+"benefits while maintaining or minimizing losses in water flows?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:26
+msgid ""
+"How will climate change and population growth impact ecosystem services "
+"and biodiversity?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:27
+msgid ""
+"In addition to secure locations for renewable energy facilities and food "
+"from fishing and aquaculture, what benefits does marine spatial planning "
+"provide to society?"
+msgstr ""
+
+#: ../../source/the_need_for.rst:30
+msgid "Introduction to InVEST"
+msgstr ""
+
+#: ../../source/the_need_for.rst:32
+msgid ""
+"InVEST is a tool for exploring how changes in ecosystems are likely to "
+"lead to changes in benefits that flow to people."
+msgstr ""
+
+#: ../../source/the_need_for.rst:34
+msgid ""
+"InVEST often employs a production function approach to quantifying and "
+"valuing ecosystem services. A production function specifies the output of"
+" ecosystem services provided by the environment given its condition and "
+"processes. Once a production function is specified, we can quantify the "
+"impact of changes on land or in the water on changes on the level of "
+"ecosystem service output."
+msgstr ""
+
+#: ../../source/the_need_for.rst:36
+msgid ""
+"InVEST uses a simple framework delineating “supply, service, and value” "
+"to link production functions to the benefits provided to people (Figure "
+"1)."
+msgstr ""
+
+#: ../../source/the_need_for.rst:43 ../../source/the_need_for.rst:120
+#: ../../source/the_need_for.rst:130
+msgid "schematic"
+msgstr ""
+
+#: ../../source/the_need_for.rst:43
+msgid ""
+"The ecosystem service supply chain, linking ecological function to "
+"ecosystem services and the benefits provided to people"
+msgstr ""
+
+#: ../../source/the_need_for.rst:45
+msgid ""
+"“Supply” represents what is potentially available from the ecosystem (ie."
+" what the ecosystem structure and function can provide). For example, "
+"this would be the wave attenuation and subsequent reduction in erosion "
+"and flooding onshore provided by a particular location and density of "
+"mangrove forest. “Service” incorporates demand and thus uses information "
```

Comment:
If a translator goes to update the `msgstr` will they be able to enter escapable (?) quotes like this?

Reply:
Normal quotes can be escaped with a backslash: `\"`. These fancy quotes `“”` are used in the source RST. Is that what you meant?